automation_tools/__init__.py

```diff
@@ -1451,20 +1467,10 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         cdn_version=sat_cdn_version,
         host=host
     )
-    # Create custom OS beaker repo
-    if os.environ.get('OS_UPGRADE_REPO'):
-        os_upgrade_repo = os.environ.get('OS_UPGRADE_REPO')
-        execute(create_custom_repos, rhel_candidate=os_upgrade_repo, host=host)
-    # Update the machine
-    execute(update_packages, host=host, warn_only=True)
 
     if distribution in ('satellite6-downstream', 'satellite6-iso'):
         execute(java_workaround, host=host)
 
-    # If using the internal dog food server's AK, unsubscribe from `CDN` first.
-    if distribution == 'satellite6-activationkey':
```

Comment:
We do not need to worry about unsubscribing since, if we are using an `activationkey`, then we won't subscribe anyhow.